extensions/msal-node-extensions/src/broker/NativeBrokerPlugin.ts

```diff
@@ -356,7 +356,7 @@ export class NativeBrokerPlugin implements INativeBrokerPlugin {
             idTokenClaims: idTokenClaims,
             accessToken: authResult.accessToken,
             fromCache: fromCache,
-            expiresOn: new Date(authResult.expiresOn * 1000),
+            expiresOn: new Date(authResult.expiresOn),
```

Comment:
Don't we always send ms? I noticed we changed this recently but do we need to document this?

Reply:
No, we return a Date object, I'm not aware that this has changed.